app/assets/javascripts/initializers/initializeVideoPlayback.js

```diff
@@ -64,6 +67,24 @@ function initializeVideoPlayback() {
     }
   }
 
+  function videoPlayerEvent(isPlaying) {
+    // jwtplayer tends to send multiple 'play' actions. This check makes sure
+    // we're not tracking repeated 'play' events for a single interaction.
+    var eventName = isPlaying ? 'play' : 'pause';
+    if (lastEvent == eventName) {
```

Comment:
I actually think it probably works fine as is, but just curious if there's a reason to _not_ use strict equality comparison here (`===` rather than `==`?)

Reply:
No reason to use it but also no reason not to use it! I included it for the sake of sticking with the better standard